packages/core/metrics/core-metrics-server-internal/src/logging/get_ops_metrics_log.ts

```diff
@@ -86,7 +86,12 @@ export function getEcsOpsMetricsLog(metrics: OpsMetrics) {
       memory: {
         heap: {
           usedInBytes: processMemoryUsedInBytes,
+          totalInBytes: process?.memory?.heap.total_in_bytes,
+          sizeLimit: process?.memory?.heap.size_limit,
         },
+        residentSetSizeInBytes: process?.memory?.resident_set_size_in_bytes,
```

Comment:
If we want these to log to stdout in pattern format, we'll need to need to fiddle with `processMemoryUsedInBytesMsg` on line 21.

```
const processMemoryUsedInBytesMsg = processMemoryUsedInBytes
    ? `memory: ${numeral(processMemoryUsedInBytes).format('0.0b')} `
    : '';
```

Reply:
<details>
  <summary>The logs are fine in `json` format:</summary>
  
  ```json
{
  "event": {
    "kind": "metric",
    "category": [
      "process",
      "host"
    ],
    "type": [
      "info"
    ]
  },
  "process": {
    "uptime": 72.65337025,
    "memory": {
      "heap": {
        "usedInBytes": 602026648,
        "totalInBytes": 695664640,
        "sizeLimit": 4345298944
      },
      "residentSetSizeInBytes": 1079312384,
      "externalInBytes": 7233984, <--- this PR
      "arrayBuffersInBytes": <--- this PR
    },
    "eventLoopDelay": 10.336272995850623,
    "eventLoopDelayHistogram": {
      "50": 10.346495,
      "95": 11.010047,
      "99": 12.140543
    },
    "eventLoopUtilization": {
      "active": 279.28487307039177,
      "idle": 4719.119918999997,
      "utilization": 0.05587480099920223
    },
    "pid": 63971
  },
  "host": {
    "os": {
      "load": {
        "1m": 6.076171875,
        "5m": 6.0078125,
        "15m": 6.32568359375
      }
    }
  },
  "service": {
    "node": {
      "roles": [
        "background_tasks",
        "ui"
      ]
    }
  },
  "ecs": {
    "version": "8.6.1"
  },
  "@timestamp": "2023-11-29T16:15:47.457-07:00",
  "message": "memory: 574.1MB uptime: 0:01:12 load: [6.08,6.01,6.33] mean delay: 10.336 delay histogram: { 50: 10.346; 95: 11.010; 99: 12.141 } utilization: 0.05587",
  "log": {
    "level": "DEBUG",
    "logger": "metrics.ops"
  },
  "trace": {
    "id": "095027dd7070903121c4316455216d32"
  },
  "transaction": {
    "id": "56fde480923f64a4"
  }
}
  ```
</details>